src/Microsoft.Sbom.Api/Utils/ComponentDetectionCliArgumentBuilder.cs

```diff
@@ -17,13 +17,18 @@ public class ComponentDetectionCliArgumentBuilder
         private string action;
         private VerbosityMode verbosity = VerbosityMode.Quiet;
         private string sourceDirectory;
-        private Dictionary<string, string> detectorArgs = new Dictionary<string, string>();
+        private Dictionary<string, string> detectorArgs = new Dictionary<string, string>() { 
+            { TimeoutArgsParamName, TimeoutDefaultSeconds.ToString() } 
```

Comment:
How does a user override this timeout?

Reply:
Users can override this by adding "Timeout={timeout in seconds}" to the list of `AdditionalComponentDetectorArgs`.